t/utils/utils.go

```diff
@@ -171,6 +173,22 @@ func GetNetworkID() int {
 	panic(fmt.Sprintf("invalid selected network: %q", *networkSelected))
 }
 
+// Deciding if test skipping is for mainnet and status chain.
+const (
+	MainnetOnly  = false
+	BothNetworks = true
+)
+
+// SkipTransactionTest is used inside tests performing transactions.
+// In case of Mainnet or StatusChain it return true to signal skipping.
+func SkipTransactionTest(skipStatus bool) bool {
```

Comment:
Wouldn't it be more flexible to declare this function with a variable number of arguments?

```go
func SkipTestInNetworks(networks... uint) bool {
  id := GetNetworkID()
  for _, network := range networks {
    if id == network {
      return true
    }
  }
  return false
}
```

Reply:
Sure, only expected there two networks so far. Maybe then even

```
func CheckTestSkipInNetworks(t *testing.T, networks ...uint) {
    ...
    if id == network {
        t.Skipf("skipping test for network %d", network)
    }
    ...
}
```